Feature/styling

am486dx opened the pull request.

am486dx committed:
LIBITD-2568. Add nav links and Banner; refactor URLs.

Added navigation links through a get_navigation_links function.
Also added the banner displaying the environment the application
is running in.

Refactored /admin/* routes into UI urls.py for cleanliness.

https://umd-dit.atlassian.net/browse/LIBITD-2568

am486dx committed:
LIBITD-2566. Updated styling.

The styling was updated to mirror Grove and track the original
IPManager more closely. I also modified how the homepage works -
the homepage now links to the group list view while the django
administration is accessable via a nav link.

This UI change is dependent on styles added to
django-umd-lib-style.

https://umd-dit.atlassian.net/browse/LIBITD-2566

peichman-umd commented on src/ipmanager/ui/templates/ui/group_list_view.html:
```diff
@@ -1,23 +1,23 @@
+<a class="button create" style="margin-bottom: .5em; margin-top: 1em" href='{% url "create_group"%}'>New Group</a>
 {% if group_list %}
 
   <table>
```
This should get the `lined` class, too.

peichman-umd commented on src/ipmanager/ui/templates/ui/edit_group.html:
```diff
@@ -31,8 +31,11 @@ <h2>Editing Group</h2>
-<button type="submit" class="edit">Update Group</button>
-<a href='{% url "single_group" group.key %}'>Cancel</a>
-<a href='{% url "list_all_groups" %}'>List All Groups</a>
+<div style="margin-top: .5em; display: flex; flex-direction: row; gap: .75em; align-items: center;">
```
This inline style on the `div` looks like it could be changed to `class="horizontal-container"`.

peichman-umd reviewed (changes requested):
Looking good. A couple changes I've called out in comments on the code. I also think it would be a good idea if you can eliminate the inline `style` attributes. Even if you create a class for a single element, in the long run that will be better for maintainability. For the more idiosyncratic and one-off a classes, my practice is to lean heavily toward a content-based name. For instance, in the archelon-django project I was styling `dl` lists in a grid layout for displaying the metadata, so I called the class `metadata`. If it turns out to be more generally useful, the name can always be updated.

am486dx committed:
LIBITD-2566. Remove inline styles.

I had a lot of cleanup to do, as I had forgotten to remove a lot
of inline styles. I also was told I shouldn't use any at all, so
I came up with replacements for the one-off styles I was using.

I also modified my use of some of the container classes to use
grids, which helped simplify the styling.

Styles are done using the Block Element Modifier (BEM) paradigm.

https://umd-dit.atlassian.net/browse/LIBITD-2566
